deploy/crds/kubernetes/kustomization.yaml

```diff
@@ -2,12 +2,14 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
- - ../common
- - monitoring.coreos.com_alertmanagerconfigs.yaml
- - monitoring.coreos.com_alertmanagers.yaml
- - monitoring.coreos.com_podmonitors.yaml
- - monitoring.coreos.com_probes.yaml
- - monitoring.coreos.com_prometheuses.yaml
- - monitoring.coreos.com_prometheusrules.yaml
- - monitoring.coreos.com_servicemonitors.yaml
- - monitoring.coreos.com_thanosrulers.yaml
+  - ../common
```

Comment:
I think it is easier to understand if the structure was something like 
```
crds
├── kustomization.yaml
├── operator
│   └── kustomization.yaml
└── rhobs-po
    └── kustomization.yaml

```
wdyt?

Reply:
:+1:  I was wondering about the naming. I didn't change things due to lack of context.